README.md

```diff
@@ -51,21 +51,21 @@ Dependencies between packages in various layers of the system are enforced via a
 [docs/PACKAGES.md](./docs/PACKAGES.md).
 
 ## Building
-
-In order to build the Fluid Framework, ensure that you have installed [Git](https://git-scm.com/downloads) and
-[Node.js](https://nodejs.org/).
-
-Note: we recommend using nvm (for [Windows](https://github.com/coreybutler/nvm-windows) or
-[MacOS/Linux](https://github.com/nvm-sh/nvm)) to install Node.js, in case you find yourself needing to install different
-versions of Node.js side-by-side.
-
-Clone a copy of the repo and change to the repo root directory:
+Ensure [Git](https://git-scm.com/downloads) is installed, then clone a copy of the git repo and change to the repo root directory:
 
 ```shell
 git clone https://github.com/microsoft/FluidFramework.git
 cd FluidFramework
 ```
 
+In order to build the Fluid Framework you will need to install the currently supported version of [Node.js](https://nodejs.org/).
+
+The recommended way to do this is to use NVM ([Windows](https://github.com/coreybutler/nvm-windows#readme) | [MacOS/Linux](https://github.com/nvm-sh/nvm#about)) and run 'nvm install' from the repo root:
+```shell
+nvm install
+```
+If choosing to install Node.js without NVM, install the [latest release](https://nodejs.org/en/download/releases/) of the version specified in the [.nvmrc file](https://raw.githubusercontent.com/microsoft/FluidFramework/main/.nvmrc).
```

Comment:
.nvmrc files are not supported by NVM for Windows, I believe.  :(  Also might be good to use the term "LTS" (long term support) instead of just "latest" since I think that's the intention?

Reply:
https://github.com/coreybutler/nvm-windows/issues/128#issuecomment-754210581

Reply:
Oh, of course it doesn't.

In that case, I'll scale back and just link the '.nvmrc' file as the source of truth.

I already felt like I was spending too much valuable README.md real-estate describing how to install node.  Alternate instructions for Cmd.exe and PowerShell.exe on Windows is too much.

...but here's how you do it:

```shell
# cmd.exe
for /F "usebackq delims=" %A in (`type .nvmrc`) do nvm install %A
```

```shell
# PowerShell
nvm use $(gc .nvmrc)
```

Reply:
(...but thanks for noticing, @markfields !)